src/sections/MeetTeam/MeetTeam.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import './MeetTeam.scss';
+import Heading from '../../components/Heading/Heading';
+import ImageCard from '../../components/ImageCard/ImageCard';
+
+
+var teamOne = require('../../assets/team1.png');
+var teamTwo = require('../../assets/team2.png');
+var teamThree = require('../../assets/team3.png');
+
+const MeetTeam = () => {
+  return (
+    <div className='meet-team'>
```

Comment:
can we use the section class here to add margin-top, instead of adding it explicitly?

Reply:
Updated